src/js/rx/containers/Active.jsx

```diff
@@ -118,11 +118,11 @@ class Active extends React.Component {
                 onChange={this.handleSort}
                 onClick={this.handleSort}
                 options={sortOptions}
-                selected={sortValue.value}/>
+                selected={sortValue}/>
```

Comment:
Why are `sortValue` and `currentSort` above both assigned with `this.props.sort`? I'm guessing it should just be `currentSort`.